src/main/java/ru/itmo/sd/deadliner2bot/model/Chat.java

```diff
@@ -0,0 +1,21 @@
+package ru.itmo.sd.deadliner2bot.model;
+
+import javax.persistence.*;
+import java.math.BigInteger;
+import java.util.Set;
+
+@Entity
+@Table(name = "chats")
+public class Chat {
+
+    @Id
+    @GeneratedValue
```

Comment:
`chat_id ` is generated by Telegram API, we don't generate it on our own. User `@GeneratedValue` only for `BIGSERIAL` types.

Reply:
Do you mean, that we do not have values, for which `@GeneratedValue` can be used?

Reply:
Only in this case, for the rest of `Id`s we will still need to generate values. `ChatId` is the only exception so far.